srml/staking/src/lib.rs

```diff
@@ -441,6 +441,15 @@ pub struct Exposure<AccountId, Balance: HasCompact> {
 	pub others: Vec<IndividualExposure<AccountId, Balance>>,
 }
 
+/// A slashing event occurred, slashing a validator for a given amount of balance.
```

Comment:
What is `own_slash`?

Reply:
the amount of slashing we've done on our own exposure

Reply:
yep, do you think it’s worthwhile to put it in the rustdoc

Reply:
well, it's not a public member, so no

Reply:
I don't see much point in documenting functions or fields *not* in rustdoc.

Reply:
(I added a comment about it last night - but it won't go in rustdoc)

Reply:
It could still be a `///` comment, without going to html docs - I don't really see any issue with that, it just helps with readability, by having consistency in where to look for description of an item.

Reply:
Ah, FWIW, I thought about mentioning it in the rustdoc of the struct in the same way `who` and `amount` was mentioned.

While I agree with @tomusdrw, I am good with status quo as well since it is improvement and we already spent enough time on this.